config/routes.rb

```diff
@@ -35,6 +35,8 @@
   get '/openid_connect/authorize' => 'openid_connect/authorization#index'
   get '/openid_connect/logout' => 'openid_connect/logout#index'
 
+  post '/usps_upload' => 'usps_upload#create'
```

Comment:
If this is an API-like endpoint not designed for users, then I'd suggest putting it in an `api/v1` namespace. I already recommended this to @mryenq for the Twilio work he's doing. That lets us separate out user-focused and machine-focused endpoints since they have different auth requirements.

Reply:
@jgsmith-usds  to add auth, Is it reasonable to just create a credential to use for this similar to whats used in `ServiceProviderController` ?